model_analyzer/config/input/config_defaults.py

```diff
@@ -16,7 +16,7 @@
 # Common defaults
 #
 
-DEFAULT_CHECKPOINT_DIRECTORY = '.'
```

Comment:
Also, we shouldn't change any code here if it is going to be cherry picked to to 21.05. Since the documentation will not be in sync with the code that we ship in the SDK container.

Reply:
All the documentation that relates to these changes needs to be modified as well (e.g. the place that you mention the checkpoints will be stored inside `checkpoints` folder.